### DISCUSSION TITLE
Deadline for great pull request for contributors

### DISCUSSION THREAD

**Discussion by Suvadip-sana** [recodehive/Stackoverflow-Analysis#423] [Q&A] [answered]
@sanjay-kv 
What is the deadline to submit the pull request for the contributors? 
30th October or 10th November?

**Comment by sanjay-kv** [comment]
we will extend to 10th Nov

**Comment by Suvadip-sana** [comment, marked as answer]
Means contributor can able to create PR until 10th Nov? 
@sanjay-kv

**Reply by sanjay-kv** [reply]
yes
